src/lib/devices/shared_memory_device_manager.js

```diff
@@ -0,0 +1,48 @@
+const Debug = require('debug');
+const fs = require('fs');
+
+let debug = Debug('taskcluster-docker-worker:devices:sharedMemoryManager');
+
+class SharedMemoryDeviceManager {
+  constructor() {
+    this.devices = [new SharedMemoryDevice("/dev/shm")];
+    this.unlimitedDevices = true;
+  }
+
+  getAvailableDevice() {
+    let devices = this.getAvailableDevices();
+    let device = devices[0];
+    device.acquire();
+
+    debug(`Device: ${device.path} acquired`);
+
+    return device;
+  }
+
+  getAvailableDevices() {
+    return this.devices;
+  }
+}
+
+class SharedMemoryDevice {
+  constructor(path) {
```

Comment:
Why does the constructor accept a `path` parameter if only `/dev/shm` is allowed?